### PR TITLE
Upgrade libcore and mini

### PR DESCRIPTION
**Pull request by nokome**
These upgrades introduce operator functions e.g. `add`, `multiply` for Mini. Operators are treated just like any other functions and handled within libcore rather than by `MiniContext`

### CODE REVIEW

**Comment by nokome** [comment]
@oliver---- can you please review and merge if acceptable

**Comment by nokome** [comment]
Also, this version of libcore includes a `sleep(seconds)` function for testing UX of cell editor when long running calls are involved.

File: src/sheet/SheetEngineAdapter.js [hunk 1]
```diff
@@ -181,11 +181,9 @@ class CellAdapter {
       this._lang = null
       let sheet = cell.getDocument()
       let type = sheet.getCellType(cell)
+      let text = cell.text()
       // HACK: this marshalling should be done somewhere else
```
Comment (oliver----) [inline src/sheet/SheetEngineAdapter.js]: Seems that you fixed this HACK

Comment (oliver----) [reply src/sheet/SheetEngineAdapter.js]: If so, you could remove this comment

**Comment by oliver----** [comment]
A little weird but consequent:
image

File: src/sheet/SheetEngineAdapter.js (hunk 1 above)
Comment (nokome) [reply src/sheet/SheetEngineAdapter.js]: I'm not sure. This was quite a quick fix and still feels temporary. I'm going to propose a change in how we deal with types, including casting, so we may deal with this comment then.

Comment (oliver----) [reply src/sheet/SheetEngineAdapter.js]: ok

**Comment by oliver----** [comment]
stencila-operators-are-functions

👍  Working for me

**Comment by nokome** [comment]
> A little weird but consequent

Yes, I suggest that we deal with this in the `FunctionUsageComponent` - don't display for operator functions.

**Pull request merged**